apps/src/templates/teacherDashboard/TeacherDashboardNavigation.jsx

```diff
@@ -0,0 +1,44 @@
+import React, {Component, PropTypes} from 'react';
+import NavigationBar from '../NavigationBar';
+
+export const teacherDashboardLinks = [
```

Comment:
this is repeated in a few places, so i think it would do well as a type, similar to how we set up types in [manageStudentsRedux](https://github.com/code-dot-org/code-dot-org/blob/8c20a1ac10fbfde6f90b4900b1abf93152188c88/apps/src/templates/manageStudents/manageStudentsRedux.js#L5-L31). maybe it would be something like:

```
const TeacherDashboardNavType = {
  MANAGE_STUDENTS: {
    label: "Manage Students",
    url: "manage_students"
  },
  ...
}
```